modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -6,6 +6,11 @@ endif::[]
 
 The Exchange backend uses REST Connect to transparently convert a REST API specification to a Mule 3 or Mule 4 connector. You can use this connector as you would any other in Anypoint Studio.
 
+[WARNING]
+====
+Mule 3 has reached its End of Life. The generation of Mule 3 connectors will be disabled in June 2025.
```

Comment:
```suggestion
Mule 3 has reached its end of life. The generation of Mule 3 connectors will be disabled in June 2025.
```

Reply:
@liocastiglioni Do we have a specific date for end of life, like June 1 or June 30?